src/Cake.Issues.PullRequests.Tfs/TfsPullRequestSystemAliases.PullRequestSystem.cs

```diff
@@ -99,6 +99,69 @@ public static partial class TfsPullRequestSystemAliases
             return context.TfsPullRequests(new TfsPullRequestSystemSettings(repositoryUrl, pullRequestId, credentials));
         }
 
+        /// <summary>
+        /// Gets an object for writing issues to Team Foundation Server or Azure DevOps pull request
+        /// where all required data is taken from the appropriate system environment variable.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="credentials">Credentials to use to authenticate against Team Foundation Server or
+        /// Azure DevOps.</param>
+        /// <returns>Object for writing issues to Team Foundation Server or Azure DevOps pull request.</returns>
+        /// <example>
+        /// <para>Report code analysis issues reported as MsBuild warnings to a TFS pull request:</para>
+        /// <code>
+        /// <![CDATA[
+        ///     ReportIssuesToPullRequest(
+        ///         MsBuildCodeAnalysis(
+        ///             @"c:\build\msbuild.log",
+        ///             MsBuildXmlFileLoggerFormat),
+        ///         TfsPullRequests(
+        ///             TfsAuthenticationNtlm()),
+        ///         @"c:\repo");
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory(PullRequestsAliasConstants.PullRequestSystemCakeAliasCategory)]
+        public static IPullRequestSystem TfsPullRequests(
+            this ICakeContext context,
+            ITfsCredentials credentials)
+        {
+            context.NotNull(nameof(context));
+            credentials.NotNull(nameof(credentials));
+
+            return context.TfsPullRequests(new TfsPullRequestSystemSettings(credentials));
+        }
+
+        /// <summary>
+        /// Gets an object for writing issues to Team Foundation Server or Azure DevOps pull request
+        /// where all required data (including authentication token) is taken from the appropriate system environment variable.
```

Comment:
I would mention here that it looks for the environment variables set by Azure DevOps / TFS build:

```
... is taken from the environment variables set by the Azure Pipelines / TFS build.
```

Reply:
Fixed.